### ISSUE TITLE
Interfacing HydroPso with GR2M

### ISSUE THREAD

**Issue by Josh-kara** [hzambran/hydroPSO#13]

**Comment by pandawoods** [comment]
Hello, did you solve your problem?

**Comment by hzambran** [comment]
No, I'm still trying to locally configure r-devel on my system to reproduce
the error.....

=====================================
"The journey is the reward" (Steve Jobs)
=====================================
Linux user #454569 -- Linux Mint user

**Comment by pandawoods** [comment]
Hello, i'm not sure if i understand you well but if your issue is this one : 

```
**> "Error in if (any(ProdStore < 0) | any(RoutStore < 0) | any(UH1 < 0) | :
> missing value where TRUE/FALSE needed
> Called from: CreateIniStates(FUN_MOD = RunModel_GR2M, InputsModel = InputsModel,
> ProdStore = RESULTS$StateEnd[1L], RoutStore = RESULTS$StateEnd[2L],
> ExpStore = NULL, UH1 = NULL, UH2 = NULL, GCemaNeigeLayers = NULL,
> eTGCemaNeigeLayers = NULL, verbose = FALSE)
> Browse[1]> Q"**
```

It means that probably you have very low value of Prodstore and Routstore, then you need to create your initialstates with function IniStates in airGR.

Your boundaries:
``
**lowerGR2M <- rep(0, times = 2)
upperGR2M <- rep(+999.99, times = 2)
startGR2M <- c(500, 2)**
``
are probably too much high, rember that those are Transformed values of parameter X1 (capacity of prod store) and X2 (capacity of routstore), you can try boundaries like this one:

``
**lowerGR2M <- rep(0, times = 2)
upperGR2M <- rep(+9.99, times = 2)**
``
Please let me know if anything...

**Comment by hzambran** [comment]
For the calibration of GR2M and other GR* model, you might want to have a look at: https://doi.org/10.5281/zenodo.3712382
